command/sub8_launch/launch/tf.launch

```diff
@@ -1,19 +1,19 @@
 <launch>
   <!-- This is not exactly right, because of weirdness in the K-matrix from camera_info for the stereo cams -->
     <node pkg="tf" type="static_transform_publisher" name="stereo_front_broadcaster"
```

Comment:
Sure, but after this, I will be very hesitant to merge stuff not using tf2. Tf has been deprecated since late 2013 (Hydro) and is pretty buggy.